
canopy_segmentation:
  chm:
    dataset_number:     _005
    clipping_height:    3
  rgb:
    dataset_number:     _005
  tree_classifier:
    target_species:     PIPA
    tree_index:         2
    min_band:           2
    max_band:           8